### DISCUSSION TITLE
What latency to expect in live view? +tips

### DISCUSSION THREAD

**Discussion by iBobik** [blakeblackshear/frigate#8950] [Ask A Question] [answered]
Hi,
I would like to use cameras for parking my RV. Ideally render something like birdview if possible, but not necessary.
![birdview](https://i.ytimg.com/vi/0ma1onwg_BM/maxresdefault.jpg)

But when using IP cameras I mostly saw latency in seconds.

**What to expect with Frigate on local wired network?
Do you have some tips how to make video near real-time?**

Dahua cameras are ok for this goal?
Bypass Frigate and stream directly from cameras?
WebRTC or some other protocol?
UDP? Disable some caches?
Some alternative libraries for custom frontend?

(I does not care about latency for detection, just live view on the tablet.)

**Comment by NickM-27** [comment]
It depends on how you are viewing the stream. WebRTC is the lowest latency and from what I can tell is perceivably instant.

**Reply by iBobik** [reply]
Thanks. I can imagine instant from Frigate to the browser, but from camera to Frigate it uses different protocol with additional latency, doesn't it?

**Reply by NickM-27** [reply, marked as answer]
rtsp is used, when I say virtually instant I can watch myself on the camera waving and it is realtime

**Reply by iBobik** [reply]
Thank you.
